src/styles/1-helpers/_variables.scss

```diff
@@ -0,0 +1,9 @@
+// colors
+$point-color: #000;
+$sec-arrow-color: green;
+$center-arrow-color: yellow;
+$min-arrow-color: blue;
+
+// general
+$mainFontSize: 30px;
+$mainLineHeight: 100%;
```

Comment:
When naming variables, you should follow the same style code for all variables.

Reply:
done